### PR TITLE
fix run numa  testcases faild 

### PR DESCRIPTION
**Pull request by tongxintest**
hello, when I test numa is problem
getpagesize() return is byte,but file /sys/devices/system/node/node1/meminfo is kb
so testcase is faild

when test_thp_compare and test_movepages togehter 
file /sys/devices/system/node/node1/meminfo  is not lockd

my modify result 
企业微信截图_16431667985086

### CODE REVIEW

**Comment by tongxintest** [comment]
@kvaneesh

File: memory/numa_test.py [hunk 1]
```diff
@@ -116,6 +117,7 @@ def test_thp_compare(self):
         """
         Test PFN's before and after offlining
         """
+        time.sleep(200)
```
Comment (PraveenPenguin) [inline memory/numa_test.py:120]: looks ugly to add sleep in test case , can we not avoid this

**Review by PraveenPenguin — COMMENTED** [review]
over all patch looks good , have one suggestion please have a look

File: memory/numa_test.py (hunk 1 above)
Comment (tongxintest) [reply memory/numa_test.py:120]: ok  ,i need learn how to locked file, wait me

Comment (Naresh-ibm) [reply memory/numa_test.py:120]: @tongxintest any update here?

**Pull request closed without merging**